projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.css

```diff
@@ -131,6 +131,13 @@
 	min-height: 26px;
 }
 
+#adminmenu .toplevel_page_site-notices .upsell_banner .banner__dismiss svg {
```

Comment:
[Not important, feel free to ignore this comment] 
CSS selectors are interpreted by the browser from right to left, in this case, the first thing the browser is going to do is to select all the SVG found in the DOM and then traverse the other selectors.

Maybe we could reduce the number of items in the selector and use something else than SVG as the first selector. We could change add a dismiss_banner class to the SVG icon so that we could have a selector like this:
`.banner_dismiss .dismiss_banner`